.drone.yml

```diff
@@ -84,6 +84,25 @@ pipeline:
       local: false
       event: [ push ]
 
+  update-search-index:
+    image: algolia/docsearch-scraper
+    entrypoint: ""
+    pull: true
+    secrets: [ 
+      algolia_app_id, algolia_config, 
+      algolia_index_name, algolia_search_api_key, 
+      algolia_write_api_key 
+    ]
+    environment:
```

Comment:
Drop most of the environment block, secrets are injected as environment variables.

Reply:
I dropped the two that seemed okay to drop.

Reply:
The others like `ALGOLIA_API_KEY`, `APPLICATION_ID` and `API_KEY` should also be dropped from environment, just define them as secrets, if it's a secret at all.

Reply:
They're not really secrets. Thanks for the suggestion.